pkg/cmd/cli/describe/describer.go

```diff
@@ -556,12 +557,32 @@ func (d *RouteDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
+	endpoints, err := d.kubeClient.Endpoints(namespace).Get(route.Spec.To.Name)
+	if err != nil {
```

Comment:
How to report a error in this case ? I think write nothing or write "<none>" is wrong.